### PR TITLE
[sketch-builder]: CLI prompt for init

### PR DESCRIPTION
**Pull request by lukasholzer**
create interactive a basic config out of the box

screenshot 2019-01-02 at 16 00 45

### CODE REVIEW

File: packages/sketch-builder/config.json [hunk 1]
```diff
@@ -1,18 +1,11 @@
 {
   "metaInformation": "../../node_modules/@sketchmine/code-analyzer/dist/meta-information.json",
   "agent": "../../node_modules/@sketchmine/dom-agent/lib/index.esm.js",
-  "host": {
-    "protocol": "http",
-    "name": "localhost",
-    "port": 4200
-  },
+  "url": "http://localhost:4200",
   "rootElement": "body > *:nth-child(2)",
```
Comment (ddprrt) [inline packages/sketch-builder/config.json]: nice nice

File: packages/sketch-builder/src/cli/questioner.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+import { questions } from './questions';
+
+interface Answer {
+  name: string;
+  answer: any;
+}
+
+export async function questioner(): Promise<{[key: string]: any}> {
+  const prompt$ = new Subject<{}>();
```
Comment (ddprrt) [inline packages/sketch-builder/src/cli/questioner.ts]: why `new Subject<{}>`

**Comment by lukasholzer** [comment]
solves a part of #36

File: packages/sketch-builder/src/cli/questioner.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+import { Subject } from 'rxjs';
+
+import { questions } from './questions';
+
+interface Answer {
+  name: string;
+  answer: any;
+}
+
+export async function questioner(): Promise<{[key: string]: any}> {
```
Comment (ddprrt) [inline packages/sketch-builder/src/cli/questioner.ts]: type AnyObject = {[key: string]: any};

**Pull request merged**